test/inductor/test_torchinductor_opinfo.py

```diff
@@ -377,6 +376,8 @@ def wrapper_set_seed(op, *args, **kwargs):
     "linalg.solve_triangular": {"check_gradient": False},
     "linalg.lu_factor": {"check_gradient": False},
     "linalg.lu_factor_ex": {"check_gradient": False},
+    # grad calculation below fails for both the aten and the compiled implementation.
+    "linalg.eig": {"check_gradient": False},
```

Comment:
Something is off here if we have to manually specify check_gradient.. It's weird that this is not working : https://github.com/pytorch/pytorch/blob/main/test/inductor/test_torchinductor_opinfo.py#L507-L515.

File an issue that the opinfo for linalg.eig (and friends) do not correctly specify either sample inputs or the supported_backward_dtypes ? Maybe worth mentioning that they're not currently exercised in `test_ops_gradients` according to the comment as well.

Reply:
Filed #109353.

> Maybe worth mentioning that they're not currently exercised in test_ops_gradients according to the comment as well.

I think the comment about `test_ops_gradients` only applies to complex32; the test is failing for the float{32,64} dtypes, I guess because float32 inputs can result in complex eigenvalues.